scenario_editor/src/Matrix/Matrix.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import './Components.css';
+import Nav from './Nav'
+import React, { Component } from 'react'
+import ReactDataSheet from 'react-datasheet';
+// Be sure to include styles at some point, probably during your bootstrapping
+import 'react-datasheet/lib/react-datasheet.css';
+
+function Matrix() {
+  return (
+    <div> 
+      <Nav/>
+      <h1>Matrix</h1>
+    </div>
+  );
+}
+
+class App extends React.Component {
```

Comment:
Id recommend against using class based components mixed with functional components like the rest of the code. It can get very confusing